ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs

```diff
@@ -149,137 +203,194 @@ void StartImportActivity(int resultConstant)
+                RootDocumentType.DeIdCardFront,
+                RootDocumentType.DeIdCardBack,
+            });
+
+            configuration.SetTopBarButtonsInactiveColor(Color.White);
+            configuration.SetTopBarBackgroundColor(Color.Black);
+
+            //configuration.SetFieldsDisplayConfiguration(
+            //    new Dictionary<string, FieldProperties>()
+            //    {
+            //        { DePassport.NormalizedFieldNames.Photo,  new FieldProperties("My passport photo", FieldProperties.DisplayState.AlwaysVisible) },
+            //        { MRZ.NormalizedFieldNames.CheckDigitGeneral,  new FieldProperties("Check digit general", FieldProperties.DisplayState.AlwaysVisible) },
+            //    }
+            //);
+
+            var intent = GenericDocumentRecognizerActivity.NewIntent(this, configuration);
+            StartActivityForResult(intent, GENERIC_DOCUMENT_REQUEST);
+        }
+
+        private void ScanVIN()
+        {
+            var configuration = new VinScannerConfiguration();
+            configuration.SetTopBarButtonsColor(Color.Gray);
+            configuration.SetTopBarBackgroundColor(Color.Black);
+
+            var intent = VinScannerActivity.NewIntent(this, configuration);
+            StartActivityForResult(intent, SCAN_VIN_REQUEST);
+        }
+
+        private void ScanEULicensePlate()
+        {
+            var configuration = new LicensePlateScannerConfiguration();
+            configuration.SetTopBarButtonsColor(Color.Gray);
+            configuration.SetTopBarBackgroundColor(Color.Black);
+
+            var intent = LicensePlateScannerActivity.NewIntent(this, configuration);
+            StartActivityForResult(intent, SCAN_EU_LICENSE_REQUEST);
+        }
+
+        private void ScanData()
+        {
+            // Launch the TextDataScanner UI
+            var step = new TextDataScannerStep(
+                 stepTag: "tag",
+                 title: string.Empty,
+                 guidanceText: string.Empty,
+                 pattern: string.Empty,
+                 shouldMatchSubstring: true,
+                 validationCallback: new ValidationCallback(),
+                 cleanRecognitionResultCallback: new RecognitionCallback(),
+                 preferredZoom: 1.6f,
+                 aspectRatio: new IO.Scanbot.Sdk.AspectRatio(4.0, 1.0),
+                 unzoomedFinderHeight: 40f,
+                 allowedSymbols: new List<Java.Lang.Character>(),
+                 significantShakeDelay: 0);
+            var config = new TextDataScannerConfiguration(step);
+
+            StartActivityForResult(TextDataScannerActivity.NewIntent(this, config), SCAN_DATA_REQUEST);
+        }
+
+        private void ScanEhic()
+        {
+            var configuration = new HealthInsuranceCardScannerConfiguration();
+            configuration.SetTopBarButtonsColor(Color.White);
+            configuration.SetTopBarBackgroundColor(Color.Black);
+            // configuration.SetFinderTextHint("custom text");
```

Comment:
@yurii-scanbot, is this comment for illustration purposes? If so, I would add another comment saying that there are additional configuration properties and then also add more commented-out properties to show them. @scan-mayank you may want to look at your own example and match the same properties.